python/cudf/cudf/_lib/column.pyi

```diff
@@ -68,9 +68,9 @@ class Column:
     @property
     def children(self) -> Tuple[ColumnBase, ...]: ...
     def set_base_children(self, value: Tuple[ColumnBase, ...]) -> None: ...
-    def _detach_refs(self, zero_copied=False) -> None: ...
-    def _has_a_weakref(self) -> bool: ...
-    def _is_cai_zero_copied(self) -> bool: ...
+    def _unlink_shared_buffers(self, zero_copied=False) -> None: ...
+    def _is_shared_buffers(self) -> bool: ...
+    def _buffers_zero_copied(self) -> bool: ...
```

Comment:
the grammar of this method name doesn't really make sense.

I think that there are two styles of sharing/referencing of the buffer:

1. trackable references (this is when cudf creates a view/shallow copy)
1. untrackable references (when the pointer is exposed through cuda-array-interface or similar)

So, with that, how about this naming?

```
def _has_trackable_references(self) -> bool:
def _has_untrackable_references(self) -> bool:
```
Or perhaps `self._internally_referenced()` and `self._externally_referenced()` ?

I realise that the colour of this bikeshed is a small part of the changes as a whole, but I kind of think picking the names will help reading the logic.

Reply:
> Or perhaps `self._internally_referenced()` and `self._externally_referenced()` ?

This seems to make it more easier to understand. Hence named it `_is_internally_referenced` & `_is_externally_referenced`